CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # (Unreleased; add upcoming change notes here)
+
 - iodide notebooks now automatically save to the server
 - fix bug where we should incorrect revision save times in history viewer,
   also show revision save time down to the second level
+- alpha now less extreme
```

Comment:
If you follow my suggestion below, you can just say something like "stop hardcoding server name in demo notebooks"

Reply:
i thought this would be a cute in-joke for us :-)

Reply:
Ok leave it in then :)